priompt/src/lib.ts

```diff
@@ -1561,9 +1559,8 @@ function computePriorityLevels(elem: AnyNode[] | AnyNode, parentPriority: number
 			levels.add(priority);
 			// we make the elem have this priority, so that we don't need to redo the priority calculation
 			elem.absolutePriority = priority;
-			// then for each child
-			for (const child of elem.children) {
-				computePriorityLevels(child, priority, levels);
+			if (Array.isArray(elem.children)) {
```

Comment:
hmm if elem.children isn't an array, shouldn't we still call computePriorityLevels on it? i may be missing something

Reply:
Good catch! I think this was a bug before as well then. Both `validateNotBothAbsoluteAndRelativePriority` and `validateNoChildrenHigherPriorityThanParent` have similarly buggy logic that also needs to be fixed.